onnxruntime/core/providers/webnn/builders/impl/normalization_op_builder.cc

```diff
@@ -114,11 +176,6 @@ bool NormalizationOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initi
     LOGS(logger, VERBOSE) << "Cannot get input shape.";
     return false;
   }
-  const auto rank = input_shape.size();
-
-  NodeAttrHelper helper(node);
-  int64_t axis = helper.Get("axis", -1);
-  axis = HandleNegativeAxis(axis, rank);
 
   const auto& scale_name = input_defs[1]->Name();
   if (!Contains(initializers, scale_name)) {
```

Comment:
@honry WebNN and ONNX both accept scale and bias as inputs from other operators, no need for tensors to be constant initialized. What are the implications of removing this check? (Zesong, this is not a blocking comment for this CR)

Reply:
@fdwr, good point! Thanks!

My understanding is this may not be a mandatory requirement, unless the implementation need to validate the concreted buffer data in these two Operand options during graph building (such case is rare). 

@zesongw, could you help verify this?

Reply:
Merging requires all conversations to be marked as resolved "1 conversation must be resolved before merging.", even though it's not smart enough to know that this comment isn't blocking for *this* CR. So we can either mark it as resolved now and address it later, or investigate it now and update accordingly in this CR?

Reply:
I have tested the Ops with and without initializer, and the results are correct.

Reply:
@zesongw, thanks for quick fixing. 

@fdwr, could you help restart the failed job?

Reply:
Cool. 😎